1.24/Dockerfile.c8s

```diff
@@ -0,0 +1,94 @@
+FROM quay.io/sclorg/s2i-base-c8s:c8s
```

Comment:
Is there any reason for building Centos Stream images on top of s2i-base instead of s2i-core?
I doublechecked and this issue is actual also for 1.20/Dockerfile.c9s and 1.22/Dockerfile.c9s.

Reply:
Good catch, fixed by https://github.com/sclorg/nginx-container/pull/278/commits/e677713face138b5810d00284e03585a99f1d657

Reply:
Thanks, could you please also fix these two: 1.20/Dockerfile.c9s and 1.22/Dockerfile.c9s? So we do not forget about it. Or should we leave them as they are, if they are already released?